### DISCUSSION TITLE
Enable oshi SystemMetrics in javaagent

### DISCUSSION THREAD

**Discussion by shravani154** [open-telemetry/opentelemetry-java-instrumentation#6540] [Q&A]
I am unable to see SystemMetrics emitted by oshi in collector when application is running as image. I am using javaagent version 1.16.0 and opentelemtry oshi version 1.16.0-alpha. Is there any configuration or property that I need to enable?

**Comment by trask** [comment]
hi @shravani154! check out and please comment on https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1566
